RHDEVDOCS-4016 Pipelines 1.7 RN revisions (#45012)
openshift/openshift-docs: modules/op-installing-pipelines-operator-using-the-cli.adoc

  source: redhat-operators <3>
  sourceNamespace: openshift-marketplace <4>
----

> <1> Specify the channel name from where you want to subscribe the Operator. The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`. In addition, the `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.

Reviewer: The first sentence mentions "subscribe." The following ones mention "install."  Is that correct?

Reviewer: ```suggestion
<1> The channel name of the Operator. Include the version number or `latest`. For example, `pipelines-1.7` or `pipelines-latest`. `latest` installs the most recent stable version of the {pipelines-shortname} Operator.
```
Parallelism. Most callouts use noun phrasing. Numbered callouts that look like numbered steps might cause confusion.
